src/app/mod.rs

```diff
@@ -335,6 +351,24 @@ pub async fn execute_files(files: Vec<PathBuf>, state: &state::AppState<'_>) ->
 
     Ok(())
 }
+async fn execute_commands(commands: Vec<String>, state: &state::AppState<'_>) -> Result<()> {
+    info!("Executing commands: {:?}", commands);
+    for command in commands {
+        exec_from_string(&command, state).await?
+    }
+
+    Ok(())
+}
+
+async fn exec_from_string(sql: &str, state: &state::AppState<'_>) -> Result<()> {
```

Comment:
this is the actual code change to support the feature -- the rest of the PR is just infrastructure and tests